### PR TITLE
Update Dependencies

### PR DESCRIPTION
**Pull request by limtingzhi**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -82,9 +81,9 @@
   },
   "lint-staged": {
     "*.{js,flow}": [
-      "lint",
-      "test",
-      "flow"
+      "yarn lint",
+      "yarn flow focus-check",
```
Comment (moroine) [inline package.json:85]: Oh I didn't know this command, what's the difference with `check`

**Review by moroine — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (limtingzhi) [reply package.json:85]: I had some issues with `lint-staged` library and based on their documentation, they recommended using `focus-check` ([link](https://github.com/okonet/lint-staged#typecheck-your-staged-files-with-flow)).

`focus-check` explanation: 
```EXPERIMENTAL: Does a focused Flow check on the input files/directories (and each of their dependents and dependencies) and prints the results.```

**Pull request merged**